lambda/readPass/index.js

```diff
@@ -84,8 +84,15 @@ exports.handler = async (event, context) => {
       }
       // Filter reservation number
       if (event.queryStringParameters.reservationNumber) {
-        queryObj.ExpressionAttributeValues[':sk'] = { S: event.queryStringParameters.reservationNumber };
-        queryObj.KeyConditionExpression += expressionBuilder('AND', queryObj.KeyConditionExpression, 'sk =:sk');
+        queryObj.ExpressionAttributeNames['#registrationNumber'] = 'registrationNumber';
```

Comment:
This is not needed, registrationNumber is not a reserved word.

Reply:
This has been updated so it no longer uses ExpressionAttributeNames.  
searchLastName, searchFirstName, and email were also updated since they also are not reserved words.